README.MD

```diff
@@ -94,6 +94,8 @@ const unlockSecret = '0x0000000000000000000000000000000000000000000000000000unlo
 const txOptionOrigin = { from: '0x0000000000000000originfacilitatoraddress' };
 const txOptionAuxiliary = { from: '0x0000000000000auxiliaryfacilitatoraddress' };
 
+// blockNumber: Block height at which stake was done. 
```

Comment:
you should probably accept the return parameters of `facilitator.stake` in the above snippet and pass them in progress take.